cms/envs/common.py

```diff
@@ -284,7 +284,7 @@
     # Special Exams, aka Timed and Proctored Exams
     'ENABLE_SPECIAL_EXAMS': False,
 
-    'ORGANIZATIONS_APP': True,
+    'ORGANIZATIONS_APP': False,
```

Comment:
Have this feature flag. configured through configurations as well instead of changing it from here.

Reply:
Yes, it has already been taken care of.

Reply:
Why is this appearing in file changes?

Reply:
Its default value was False, I changed it earlier, now reverting back.
Secondly, it was the reason why I wanted to create a separate PR.

Reply:
okaa

Reply:
@HamzaIbnFarooq sorry I did not get it, why this file shows changes?

Reply:
@tasawernawaz In a previous PR, I changed ORGANIZATIONS_APP flag to True. But now Devops changed this variable in lms.env.json, so I just changed the variable in code to default. It doesn't make any effect.

Reply:
ohk